<launch>
  <arg name="joy_config" default="logitechF310" />
  <arg name="joy_dev" default="/dev/input/js2" />
  
  <node pkg="joy" type="joy_node" name="joy_node">
    <param name="dev" value="$(arg joy_dev)" />
    <param name="deadzone" value="0.03" />
    <param name="autorepeat_rate" value="20" />
  </node>

  <node pkg="taurob_teleop_twist_joy" name="teleop_twist_joy" type="teleop_node" output="screen">
    <rosparam command="load" file="$(find taurob_teleop_twist_joy)/config/$(arg joy_config).config.yaml" />
  </node>
</launch>
